bob-api/api/de_onderwijzers_api.php

```diff
@@ -0,0 +1,195 @@
+<?php namespace api;
+
+use ErrorException;
+
+/**
+ * Class DeOnderwijzers
+ *
+ * Wrapper for de API die de onderwijzers beschikbaar hebben gesteld.
+ *
+ * Documentatie die goed is om te lezen:
```

Comment:
Is het niet beter om dit in de read.me te zetten?

Reply:
Zeker, ga ik doen!